src/app/products/components/product-delete-confirm-dialog/product-delete-confirm-dialog.component.html

```diff
@@ -0,0 +1,8 @@
+<h1 mat-dialog-title>Please confirm</h1>
+<div mat-dialog-content>
+    <p>Are you sure you want to delete {{ product.name }}?</p> 
+</div>
+<div mat-dialog-actions class="dialog-actions">
+    <button mat-button (click)="onCancelClick()">Cancel</button>
```

Comment:
Methods called from the Angular templates should be named by what they do, not by how they are called.

Right now, it is hard to tell what will happen when the button is clicked, making the template hard to read in isolation.
Naming the handler method by what it does makes it easier to reason about the template:
`<button (click)="cancelDeletion()">Cancel</button>`
This makes templates, tests, and the code itself easier to read.